cirq/aqt/__init__.py

```diff
@@ -12,6 +12,12 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from cirq.aqt.aqt_sampler import AQTSampler, AQTSamplerLocalSimulator
-from cirq.aqt.aqt_device import AQTSimulator
+from cirq.aqt.aqt_sampler import (
+    AQTSampler,
+    AQTSamplerLocalSimulator,
+)
+
+from cirq.aqt.aqt_device import (AQTSimulator, AQTNoiseModel, get_aqt_device,
+                                 get_aqt_default_noise_dict,
+                                 get_aqt_device_op_string)
```

Comment:
This formatting looks a little goofy.  Maybe try adding a trailing comma at the end of get_aqt_device_op_string to see if it fixes things?